README.md

```diff
@@ -208,11 +208,18 @@ All relevant commits will be signed with the key `0x24BB6BF4` for additional sec
 
 ### Development and contributing
 
-We rely on npm-run-scripts for integrating with out tooling infrastructure. We use ESLint as a pre-commit hook to ensure code consistency. Morover, to ease formatting we use [prettier](https://github.com/prettier/prettier) while building the `/dist` assets happens through `rollup`.
+#### Installation (`yarn i`)
 
-These are our npm scripts
+We support both `yarn` and `npm@5.2` officially while providing lock-files for either dependency manager to provide reproducible installs and builds on either or. TravisCI itself is configured to install dependencies using `yarn`. When using an older version of `npm` we can not fully ensure the versions of installed dependencies which might lead to unanticipated problems.
```

Comment:
nit: slightly better wording might be...
--
+We support both `yarn` and `npm@5.2` officially, while providing lock-files for either dependency manager, to provide reproducible installs and builds. TravisCI is configured to install dependencies using `yarn`. When using an older version of `npm` the versions of installed dependencies cannot be insured, which may lead to unanticipated problems.